src/log/leveldb.cpp

```diff
@@ -156,7 +158,10 @@ static string encode(uint64_t position, bool adjust = true)
 LevelDBStorage::LevelDBStorage()
   : db(nullptr), first(None())
 {
-  // Nothing to see here.
+  Option<string> disableAutoCompactVar = os::getenv("MESOS_LOG_AUTO_COMPACT_DISABLED");
+  if (autoCompactVar.isSome() && autoCompactVar.get() == "1") {
+    autoCompact = false;
+  }
```

Comment:
the names don't line up anymore? we can use getOrElse here too:

```suggestion
  // We provide an escape hatch in case some users experience issues
  // with auto-compaction. This is currently only available as an environment
  // variable since it's easier than exposing a flag up through the JNI layer.
  Option<string> disable = os::getenv("MESOS_LOG_AUTO_COMPACT_DISABLED");
  if (disable.getOrElse("0") == "1") {
    autoCompact = false;
  }
```